### DISCUSSION TITLE
attempt to load Julia "Symbolics.jl" package failed

### DISCUSSION THREAD

**Discussion by TimFH** [sagemathinc/cocalc#6286] [Q&A]
Why did this fail?

[error.txt](https://github.com/sagemathinc/cocalc/files/10281566/error.txt)

**Comment by haraldschilly** [comment]
The stacktrace indicates Julia wants to modify a global file, where the package is installed originally. Global files are read-only in any multi-user system. Therefore, Julia is not able to modify that file. Looks like one of those cases, where Julia is just not able to work in a multi-user environment. What you could do is define your own depot and/or project environment, which is independent of the global one. How exactly, I don't know. After years, Julia's package management is still a mystery to me.

I also see this is for julia 1.7. Depending on which line of project software environments you use, there is or will be a 1.8 version available. Maybe with a newer version, the package is more in line and this problem goes away.
